h2/src/test/org/h2/test/db/TestLinkedTable.java

```diff
@@ -714,4 +715,27 @@ private void testGeometry() throws SQLException {
         ca.close();
         cb.close();
     }
+    
+    private void testFetchSize() throws SQLException {
+        if (config.memory && config.mvStore) {
+            return;
+        }
+        org.h2.Driver.load();
+        Connection ca = DriverManager.getConnection("jdbc:h2:mem:one", "sa", "sa");
+        Connection cb = DriverManager.getConnection("jdbc:h2:mem:two", "sa", "sa");
+        Statement sa = ca.createStatement();
+        Statement sb = cb.createStatement();
+        sa.execute("DROP TABLE IF EXISTS TEST; "
+                + "CREATE TABLE TEST as select st_makepoint(-60 + n*random()/500.00, 30 + n*random()/500.00), n as id from generate_series(1,1000) as n;");
```

Comment:
You can't use `ST_MAKEPOINT` in raw H2.

Please, also don't use undocumented and unsupported outside of PostgreSQL compatibility mode `generate_series` in H2, use `SYSTEM_RANGE`.